src/libs/actions/ReimbursementAccount/setupWithdrawalAccount.js

```diff
@@ -58,7 +58,7 @@ function getNextStep(updatedACHData) {
         return currentStep;
     }
 
-    if (currentStep === CONST.BANK_ACCOUNT.STEP.VALIDATION && updatedACHData.bankAccountInReview) {
```

Comment:
Did you make sure no other places used `bankAccountInReview`?

Reply:
Oh I need to undo this. We actually removed `bankAccountInReview` completely from the web/auth side. 

And looking at this page, I don't think this method is used anymore? Its used in `setupWithdrawalAccount()`, but that is no longer being used I don't think.

Reply:
I'm going to undo this change

Reply:
setupWithdrawalAccount() isn't being used anymore, created an issue to remove this file completely: https://github.com/Expensify/Expensify/issues/240702